diff --git a/node_modules/@fortawesome/fontawesome-common-types/index.d.ts b/node_modules/@fortawesome/fontawesome-common-types/index.d.ts
index b70b000..affaf9b 100644
--- a/node_modules/@fortawesome/fontawesome-common-types/index.d.ts
+++ b/node_modules/@fortawesome/fontawesome-common-types/index.d.ts
@@ -1,4 +1,4 @@
-export type IconPrefix = "fas" | "fab" | "far" | "fal" | "fad";
+export type IconPrefix = "fas" | "fab" | "fac" | "far" | "fal" | "fad";
 export type IconPathData = string | string[]
 
 export interface IconLookup {
@@ -21,7 +21,28 @@ export interface IconPack {
   [key: string]: IconDefinition;
 }
 
-export type IconName = '500px' | 
+export type IconName = 'antdesign-icon' |
+  'apollo-icon' |
+  'auth-icon' |
+  'cloudinary-icon' |
+  'crapp-icon' |
+  'gatsby-icon' |
+  'graphql-icon' |
+  'heroku-icon' |
+  'jwt-icon' |
+  'mongodb-icon' |
+  'mysql-icon' |
+  'next-icon' |
+  'oauth-icon' |
+  'postgresql-icon' |
+  'redux-icon' |
+  'styledcomponents-icon' |
+  'stylelint-icon' |
+  'vercel-icon' |
+  'vim-icon' |
+  'tailwind-icon' |
+  'typescript-icon' |
+  '500px' | 
   'accessible-icon' | 
   'accusoft' | 
   'acquisitions-incorporated' | 
